addons/web/static/src/core/emoji_picker/emoji_picker.scss

```diff
@@ -22,15 +23,26 @@
         .o-Emoji {
             filter: grayscale(1);
         }
+
+        background-color: var(--o-EmojiPicker__navbar-active, $gray-100) !important;
```

Comment:
Following BEM notation, states should be preceded by "--" -> --o-EmojiPicker__navbar--active 😉